### PR TITLE
Delete unused packages

### PR DESCRIPTION
**Pull request by jasonmalinowski**
Delete some stuff we're not using anymore.

### CODE REVIEW

**Comment by jaredpar** [comment]
Looks like most of our progression unit tests are failing. Appears they try and dynamically load that DLL. Are the tests necessary or they just more artifacts laying around that we don't need anymore?

**Comment by jasonmalinowski** [comment]
We wouldn't want to get rid of those tests. I'll have to dig further then.

**Comment by jasonmalinowski** [comment]
Alright fixed up so this fails less. @dotnet/roslyn-ide or @dotnet/roslyn-infrastructure just a cleanup change.

File: src/Tools/GenerateSdkPackages/Microsoft.VisualStudio.Debugger.Engine.nuspec [hunk 1]
```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0"?>
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
```
Comment (genlu) [inline src/Tools/GenerateSdkPackages/Microsoft.VisualStudio.Debugger.Engine.nuspec]: what are those deleted nuget packages?

Comment (jasonmalinowski) [reply src/Tools/GenerateSdkPackages/Microsoft.VisualStudio.Debugger.Engine.nuspec]: @genlu These were packages we were creating of Visual Studio assemblies we didn't own for our own reference purposes. The debugger now creates actual packages we use, so we don't need these anymore.

**Pull request merged**